test/integration/server/inspec/server_spec.rb

```diff
@@ -5,3 +5,27 @@
 describe iptables do
   it { should have_rule('-A smtp -p tcp -m tcp --dport 25 -j ACCEPT') }
 end
+
+describe package 'postfix-perl-scripts' do
+  it { should be_installed }
+end if os.family == 'redhat'
+
+describe file '/usr/local/bin/pfcat' do
+  it { should exist }
+  it { should be_executable }
+  its('content') { should match /# pfcat - Helper script to print out mail from the Postfix queues./ }
+end
+
+describe command '/usr/local/bin/pfcat' do
+  its('stdout') { should match /Usage: pfcat <queue id>/ }
+end
+
+describe file '/usr/local/bin/pfdel' do
+  it { should exist }
+  it { should be_executable }
+  its('content') { should match /# pfdel - Deletes message containing specified address from the Postfix queue./ }
+end
+
```

Comment:
I recommend you add the following tests as well (note: I'm not including the full path):
```
# should return 1
pfcat foo
# should return 255
pfdel foo@bar.com
```